posts/app.py

```diff
@@ -96,6 +115,20 @@ def generate_timestamp() -> str:
     return "2017-10-06T00:00:00+00:00"
 
 
+def upload_file_to_cloud_and_get_url(file):
+    """Uploads a file to the GCloud Storage bucket.
+
+    Returns:
+        str: Public URL of the file in the cloud.
+    """
+    storage_client = storage.Client()
+    bucket = storage_client.get_bucket(GCLOUD_STORAGE_BUCKET_NAME)
```

Comment:
Extract this into something that you can store in a global like the DB?

Reply:
Done https://github.com/knative-portability/large-events/pull/20/commits/5615d00b24da42270add718a294db18cfb8d9170